src/pollingManager.ts

```diff
@@ -20,16 +20,16 @@ export class PollingManager {
     const jobs = await this.taskManager.getJobsByTaskStatus();
     const expirationDate = new Date();
     expirationDate.setDate(expirationDate.getDate() + this.expirationDays);
-    if ((jobs.completedJobs?.length ?? 0) > 0) {
+    if (jobs.completedJobs?.length) {
```

Comment:
fix lint

Reply:
It's ok. we don't want to handle nullish/zero/NaN cases explicitly

Reply:
but this can cause a bug as you removed the > 0 condition

Reply:
`jobs.completedJobs.length` can either be a number (including 0) **OR** `undefined`.
If it's not a positive number (i.e `undefiend` or the value `0`) we should not enter the if statement.
And that's exactly what happens here